client/customer-effort-score-tracks/customer-effort-score.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './customer-effort-score.scss';
+
+/**
+ * Use `CustomerEffortScore` to gather a customer effort score.
+ *
+ * NOTE: This should live in @woocommerce/customer-effort-score to allow
+ * reuse.
```

Comment:
Will you create a new package or add it to `@woocommerce/components`?

Reply:
Create a new package, per https://woostartp2.wordpress.com/2020/09/25/customer-effort-score-2/#comment-890